src/Foundation/Connect/code/Wooli.Foundation.Connect.csproj

```diff
@@ -598,6 +597,11 @@
     <Compile Include="Mappers\Resolvers\MultilistFieldResolver.cs" />
     <Compile Include="Models\Catalog\StockStatus.cs" />
     <Compile Include="Models\Payment\PaymentClientTokenResult.cs" />
+    <Compile Include="Models\Storefront\CountryRegionConfigurationModel.cs" />

```

Comment:
`Models\Settings\`

Reply:
All the settings should have `Settings` postfix

Reply:
This will be moved to `Models\Extensions\Storefront` in next commit

Reply:
This models are from Commerce.Generated